models/recurly__account_daily_overview.sql

```diff
@@ -44,7 +44,7 @@ final as (
         account_overview.account_state,
         account_overview.account_username 
 
-        {{ fivetran_utils.persist_pass_through_columns('recurly_account_pass_through_columns', identifier='account_history') }},
+        {{ fivetran_utils.persist_pass_through_columns('recurly_account_pass_through_columns', identifier='account_overview') }},
```

Comment:
The identifier needs to be the alias used within the relevant cte. For this CTE the alias is `account_overview` so we need to have that be listed. Essentially the identifier prepends the field with the alias. So if I have the following variable config:
```yml
  recurly_account_pass_through_columns:
    - name: cc_emails
      alias: people_who_see_my_emails
      transform_sql: cast(people_who_see_my_emails as string)
```

Then it would be listed as such in the above CTE:
```sql
      account_overview.account_state,
      account_overview.account_username,
      account_overview.people_who_see_my_emails,
      etc...

```

Reply:
Yeah that was a miss by me. Understand the logic.